server/migrations/1613157291778_post-import-cleanup.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-len */
+/* eslint-disable quotes */
+const { dbClient } = require('../db/db.js');
+const { collections } = require('../db/schema.js');
+
+exports.up = async () => {
+  await dbClient.db.withTransaction(async (tx) => {
+    await tx.query(`UPDATE ${collections.EMPLOYER_SITES} SET body = body - 'earlyAdopterAllocation' || jsonb_build_object('phaseOneAllocation', body->'earlyAdopterAllocation')`);
```

Comment:
Did a little reading. This looks good! Relevant docs [here](https://www.postgresql.org/docs/9.5/functions-json.html).